webapp-django/crashstats/supersearch/jinja2/supersearch/macros/date_filters.html

```diff
@@ -1,12 +1,17 @@
 {% macro date_filters(start_date, end_date) %}
+  {# NOTE(willkg): the start_date and end_date here are probably not in UTC #}
```

Comment:
I need to change this note to "This assumes the start_date and end_date are in UTC."